types/HasteMap.js

```diff
@@ -62,8 +63,9 @@ export type ModuleMetaData = [Path, /* type */ number];
 export type HType = {|
   ID: 0,
   MTIME: 1,
-  VISITED: 2,
-  DEPENDENCIES: 3,
+  SIZE: 2,
+  VISITED: 3,
+  DEPENDENCIES: 4,
```

Comment:
further down this file there's `HTypeValue` type, can you adjust it to:
```jsx
export type HTypeValue = $Values<HType>
```
as it got out of sync.

Reply:
done